### PR TITLE
bunny_helper switch position

### PR DESCRIPTION
**Pull request by kevthehermit**
Allows you to use $switchposition in payloads. Useful for building paths relative to payload/

### CODE REVIEW

**Comment by IMcPwn** [comment]
@kevthehermit Shouldn't it follow the format of the other variables of all caps?
SWITCH_POSITION perhaps?

**Comment by IMcPwn** [comment]
You added a whole different module. You might want to rebase everything that isn't in bunny_helpers.sh.

**Comment by kevthehermit** [comment]
Yeah just spotted that after i pushed the commit.

File: payloads/library/bunny_helpers.sh [hunk 1]
```diff
@@ -17,3 +17,26 @@ export TARGET_HOSTNAME=$(cat $leasefile | grep hostname | awk '{print $2 }' \
...
+# Taken from bash_bunny.sh
+################################################################################
+
+check_switch() {
+	switch1=`cat /sys/class/gpio_sw/PA8/data`
+	switch2=`cat /sys/class/gpio_sw/PL4/data`
+	switch3=`cat /sys/class/gpio_sw/PL3/data`
+	echo "--- switch1 = $switch1, switch2 = $switch2, switch3 = $switch3"
+	if [ "x$switch1" = "x0" ] && [ "x$switch2" = "x1" ] && [ "x$switch3" = "x1" ]; then
+		SWITCH_POSITION="switch1"
```
Comment (IMcPwn) [inline payloads/library/bunny_helpers.sh]: Shouldn't the exports be done here?

File: payloads/library/bunny_helpers.sh [hunk 2]
```diff
@@ -17,3 +17,26 @@ export TARGET_HOSTNAME=$(cat $leasefile | grep hostname | awk '{print $2 }' \
...
+	elif [ "x$switch1" = "x1" ] && [ "x$switch2" = "x0" ] && [ "x$switch3" = "x1" ]; then
+		SWITCH_POSITION="switch2"
```
Comment (IMcPwn) [inline payloads/library/bunny_helpers.sh]: and here... etc. instead of at the bottom?

**Pull request merged**